Python/Python_news_reader/Readme.md

```diff
@@ -0,0 +1,19 @@
+# -Python News Reader
+[![forthebadge made-with-python](http://ForTheBadge.com/images/badges/made-with-python.svg)](https://www.python.org/)
+
+This python program fetches the news from Indain sources and speaks or reads the news with voice over feature. A python script which will fetch news from newsapi.com and will speak that news with headlines and description. Reading news and articles sometimes may too time consuming and boring ,With the help of this python news reader you listen to top news while working on your other work.
+### -What is News API?
+News API is a simple HTTP REST API for searching and retrieving live articles from all over the web. Visit the newsapi.org for documentaion and your personal API key.
+
+
+## -Steps to run this script on your local machine:
+#### 1. Open the requirements.md and install modules that mentioned in the file.
+#### 2. Now, clone this folder 'Rotten-Scripts/Python/Python_news_reader' in your local machine.
+#### 3. Visit newsapi.org and copy your API key from there.
+#### 4. Open the file 'Url.txt' from 'Rotten-Scripts/Python/Python_news_reader' and paste your API key in the last of the link (i.e in the place of "Paste_your_API_key_here") and        save the "Url.txt" file by pressing ctrl+s command.
+#### 5. Now, open the "Python news reader.py" file and enjoye listening the top news from all over the world.

```

Comment:
Correct the Spell Mistake of `enjoye`.

Also you can mention steps like this: 

```
- Step 1
- Step 2
- Step 3
```

Rather than mentioning them as H4 Headers

Reply:
Done!